winforms/tests_backend/widgets/properties.py

```diff
@@ -11,7 +11,7 @@ def toga_color(color):
     if color in {Color.Empty, SystemColors.Control}:
         return TRANSPARENT
     else:
-        return rgba(color.R, color.G, color.B, color.A / 255)
+        return rgba(color.R, color.G, color.B, round(color.A / 255, 2))
```

Comment:
This is unrelated to the webview issue; however, CI was returning an intermittent failure because the alpha channel was having precision issues with alpha values of 0.0.

Reply:
Here's the error:
```
_________________________________ test_color __________________________________
Traceback (most recent call last):
  File "D:\a\toga\toga\testbed\build\testbed\windows\app\src\app\tests\widgets\properties.py", line 355, in test_color
    assert_color(probe.color, color)
  File "D:\a\toga\toga\testbed\build\testbed\windows\app\src\app\tests\assertions.py", line 25, in assert_color
    assert (actual.r, actual.g, actual.b, actual.a) == (
AssertionError: assert (50, 128, 200, 0.0) == (50, 128, 200, 1 � 3.9e-03)
  At index 3 diff: 0.0 != 1 � 3.9e-03
  Full diff:
  - (50, 128, 200, 1 � 3.9e-03)
  ?                ^^^^^ --- -
  + (50, 128, 200, 0.0)
  ?                ^
```

In [yesterday's run](https://github.com/beeware/toga/actions/runs/5306068919/jobs/9603574747?pr=1989), it failed with exactly the same error in multilinttextinput, numberinput, passwordinput, switch and textinput.

What this indicates is that it was expecting an alpha of 1, but it got a value of 0. 3.9e-03 is just the expected accuracy of 1/255. So this is more serious than a precision issue, and this change won't fix it.

I've seen this locally a few times, but I have no idea what's causing it.

Reply:
Since this change won't fix the problem, I suggest we revert it before merging this PR.

Reply:
Huh... 1/255 is smaller than I thought :-) 

Agreed this fix is a red herring; reverted.

Reply:
Created #2000 to track this.